client/controllers/service.go

```diff
@@ -17,3 +19,9 @@ type BookApiController interface {
 	UpdateBook(u *pb.BookItem) (*pb.UpdateBookResponse, error)
 	DeleteBook(id uint32) (*pb.DeleteBookResponse, error)
 }
+
+type BookHandlerApiControllers interface {
```

Comment:
How about `BookApiController`?

Reply:
The same mistake, it must be BookHistoryApiContrillers. Fixed